options/internal/x86_64/setjmp.S

```diff
@@ -31,3 +31,13 @@ longjmp:
 	mov 0x30(%rdi), %rsp
 	jmp *0x38(%rdi)
 
+.global sigsetjmp
+.type sigsetjmp, "function"
+sigsetjmp:
+	ret
```

Comment:
Here we either need a proper implementation, or we need to warn that it is not implemented.